atproto/identity/did.go

```diff
@@ -13,26 +14,62 @@ import (
 	"github.com/bluesky-social/indigo/atproto/syntax"
 )
 
-// WARNING: this does *not* bi-directionally verify account metadata; it only implements direct DID-to-DID-document lookup for the supported DID methods, and parses the resulting DID Doc into an Identity struct
+// Variant of ResolveDID which parses in to a DIDDocument struct.
```

Comment:
this is `ResolveDID` not a 'variant'? the think-o was that this is the variant of resolveDIDBytes() that does unmarshal? Probably:
```
// ResolveDID fetches a DIDDocument
// No further resolution (e.g. handle resolution) within the DIDDocument is performed
```

Reply:
good catch! mangled copy/paste